### PR TITLE
Disable obs browser pages for some custom DCs

### PR DESCRIPTION
**Pull request by n-h-diaz**
Updates 
* climate_trace
* custom 
* feedingamerica
* iitm
* unsdg

This is to remove dependency on BQ (and is currently broken for many instances currently), so we don't have to keep as many BQ versions

Verified locally for each instance, but wanted to double check that this won't negatively impact any prod instance?

### CODE REVIEW

**Comment by n-h-diaz** [comment]
Updated with a new field in the config based on recommendations and verified locally. Let me know if this seems okay (or if there's a better/preferred way to do this)!

File: server/app_env/_base.py [hunk 1]
```diff
@@ -91,3 +91,6 @@ class Config:
   MIN_STAT_VAR_GEO_COVERAGE = 10
   # NL Bad words file.
   BAD_WORDS_FILE = 'nl_bad_words.txt'
+  # Whether to disable BigQuery from instance. This is primarily used for
+  # accessing the observation browser pages.
+  DISABLE_BQ = False
```
Comment (keyurva) [inline server/app_env/_base.py:96]: Is it possible to make it disabled by default and enable it only for main dc?

Also, if we go with that, consider naming it `ENABLE_BQ`.

Comment (n-h-diaz) [reply server/app_env/_base.py]: Good idea, thanks! I switched to ENABLE_BQ and only added for main dc + test environments

File: server/app_env/_base.py [hunk 2]
```diff
@@ -91,3 +91,6 @@ class Config:
   MIN_STAT_VAR_GEO_COVERAGE = 10
   # NL Bad words file.
   BAD_WORDS_FILE = 'nl_bad_words.txt'
+  # Whether to enable BigQuery from instance. This is primarily used for
```
Comment (keyurva) [inline server/app_env/_base.py:94]: nit: The comment does not look right. Can you correct it?

File: server/app_env/feedingamerica.py [hunk 3]
```diff
@@ -26,4 +26,4 @@ class Config(_base.Config):
 class LocalConfig(local.Config):
   CUSTOM = True
   NAME = "Feeding America"
-  SHOW_DISASTER = False
\ No newline at end of file
+  SHOW_DISASTER = False
```
Comment (keyurva) [inline server/app_env/feedingamerica.py:29]: nit: This file can be left as is.

**Review by keyurva — APPROVED** [review]
Very cool - thanks Natalie!

File: server/app_env/_base.py (hunk 2 above)
Comment (n-h-diaz) [reply server/app_env/_base.py]: Done

File: server/app_env/feedingamerica.py (hunk 3 above)
Comment (n-h-diaz) [reply server/app_env/feedingamerica.py]: Done

**Pull request merged**